includes/class-ajax.php

```diff
@@ -756,70 +753,64 @@ public function handle_frontend_submission() {
 
         $entry_fields = apply_filters( 'weforms_before_entry_submission', $entry_fields, $form, $form_settings, $form_fields );
 
-        $entry_id = weforms_insert_entry( [
-            'form_id' => $form_id,
-        ], $entry_fields );
-
+        $entry_id = 1;

```

Comment:
To confirm, all this code is from Mahesh?

I'm not sure why we're setting `$entry_id = 1;` Isn't this the problem, the id is always 0 or 1?